annotation-error-decoder/pom.xml

```diff
@@ -50,4 +50,13 @@
       <scope>test</scope>
     </dependency>
   </dependencies>
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.moditect</groupId>
```

Comment:
Move to the top pom, and skip plugin on the one project that can't be excuted

Reply:
OK. This adding the moditect plugin to the parent, enabled by default, skip on no modular projects, right?